Symfony/CS/Tests/Fixer/Contrib/Php4ConstructorFixerTest.php

```diff
@@ -981,4 +981,51 @@ function __construct($foo)
 
         $this->makeTest($expected, $input);
     }
+
+    public function testInfiniteRecursion()
+    {
+        $expected = <<<'EOF'
+<?php
+    class Parent1
+    {
+        function _construct()
+        {
+            echo "foobar";
+        }
+    }
+
+    class Class1 extends Parent1
+    {
+        function __construct($foo)
+        {
+            parent::__construct();
+            echo "something";
+        }
+    }
+?>
+EOF;
+
+        $input = <<<'EOF'
+<?php
+    class Parent1
+    {
+        function _construct()
```

Comment:
Definitely not. The definition of the parent class is just decorative here, but still worth fixing.